application/src/Service/Delegator/FormSelectDelegatorFactory.php

```diff
@@ -10,6 +10,8 @@ public function __invoke(ContainerInterface $container, $name,
         callable $callback, array $options = null
     ) {
         $formSelect = $callback();
+        $formSelect->addTranslatableAttribute('aria-label');
+        $formSelect->addTranslatableAttribute('title');
```

Comment:
`title` is a default translatable attribute defined in the Laminas abstract form view helper so I don't believe this line is needed